DependencyInjection/Configuration.php

```diff
@@ -38,6 +39,30 @@ public function getConfigTreeBuilder()
                     ->defaultValue([
                         'controller' => ['../src/*Bundle/Controller', '../src/*Bundle/Action'],
                         'command' => ['../src/*Bundle/Command'],
+                        'event_subscriber' => ['../src/*Bundle/EventSubscriber'],
+                    ])
+                ->end()
+                ->arrayNode('tags')
+                    ->info('List of tags to add when implementing the corresponding class.')
+                    ->useAttributeAsKey('class')
+                    ->prototype('array')
+                        // Converts 'console.command' to ['console.command']
+                        ->beforeNormalization()->ifString()->then(function ($v) { return [$v]; })->end()
+                        ->prototype('array')
+                            // Converts 'console.command' to ['console.command', []]
+                            ->beforeNormalization()->ifString()->then(function ($v) { return [$v, []]; })->end()
+                            ->validate()
+                                ->ifTrue(function ($v) {
+                                    return count($v) !== 2 || !is_string($v[0]) || !is_array($v[1]);
+                                })
+                                ->thenInvalid('Invalid tag format. They must be as following: [\'console.command\', [\'foo\' => \'bar\']]')
```

Comment:
Maybe can you change `'foo' => 'bar'` by something more explicit like `'attribute => 'value'`?